README.md

```diff
@@ -1,2 +1,82 @@
-# redis-el6-rpm
-RPM Spec for Redis 3.0.x
+# REDIS-3.0.7 RPM built for RHEL 6.5
```

Comment:
@olivermel Might as well make this 6.x, since it works on all versions of RHEL 6.